src/controllers/image.controller.ts

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { Request, response, Response } from "express";
+import imageService, { ImageService } from "../services/image.service";
+import sharp from "sharp";
+import { matchedData } from "express-validator";
+
+export class ImageController {
+  constructor(private imageService: ImageService) {}
```

Comment:
you can add `readonly` here - `private readonly  imageService: ImageService`, since we don't want to modify imageService after injection